composite/outlier-detection/src/source.ts

```diff
@@ -0,0 +1,48 @@
+import { Execute } from '@chainlink/types'
+import * as GenesisVolatility from '@chainlink/genesis-volatility-adapter'
+import * as XBTO from '@chainlink/xbto-adapter'
+
+export enum SourceDataProvider {
+  XBTO = 'xbto',
+  GenesisVolatility = 'genesisvolatility',
+}
+
+const isSourceDataProvider = (envVar?: string): envVar is SourceDataProvider =>
+  Object.values(SourceDataProvider).includes(envVar as any)
+
+export const getSourceDataProviders = (): SourceDataProvider[] => {
+  const sourceDataProviders = (process.env.SOURCE_ADAPTERS || '').split(',')
```

Comment:
This should also be part of the config.

We can extend the config type to be:

```javascript
// config.ts
import { SourceDataProvider } from './source'

export type Config = types.Config & {
   sourceDataProviders: SourceDataProvider[]
   threshold: {
     checks: number
     onchain: number
   }
}
```

And then we can read it from the env in the `makeConfig` factory function.